functions/__init__.py

```diff
@@ -73,14 +68,21 @@ def update_image_order(user_code, filename, new_index):
 
 def get_user_code(request):
     user_code = request.cookies.get('user_code')
-    if not user_code:
+    conn = sqlite3.connect('uploads/database.db')
+    c = conn.cursor()
+    
+    if user_code:
+        c.execute("SELECT name FROM sqlite_master WHERE type='table' AND name=?", (user_code,))
+        exists = c.fetchone()
+    else:
+        exists = False
```

Comment:
1. IMO, the readability is better if you bring the `exists` variable to a higher level, such that it is clear that it is always declared
2. It appears that you use `exists` as a boolean later on. It would, thus, make sense to make it boolean in the first place.

In other words,
```suggestion
    exists = False
    if user_code:
        c.execute("SELECT name FROM sqlite_master WHERE type='table' AND name=?", (user_code,))
        exists = c.fetchone() is not None
```